web/packages/plugins/apps/dmt-app/src/pages/SearchPage.tsx

```diff
@@ -424,16 +437,17 @@ export default ({ settings }: any) => {
   const [searchSettings, setSearchSettings] = useLocalStorage(
     'searchSettings',
     {
-      dataSource: '',
+      dataSource: [],
```

Comment:
rename to dataSource**s**? (since it's now a list)

Reply:
fixed!